src/Congo.WebApi/Controllers/CartController.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Congo.Contracts.Responses.Cart;
+using Congo.WebApi.Data.CartAccess;
+using Mapster;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Congo.WebApi.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class CartController : ControllerBase
+    {
+        private readonly IMediator _mediator;
+
+        public CartController(IMediator mediator) => _mediator = mediator;
+
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<CartResponse>>> Get()
+        {
+            var carts = await _mediator.Send(new GetCartsQuery());
```

Comment:
I think the user can only have one cart at a time since it is persisted across all devices